blog/content/post/2023-09-27-aem_phase2.md

```diff
@@ -57,7 +57,7 @@ without DRTM.
 ### Support for TPM 2.0 module in Xen
 
 With the introduction of TPM 2.0 support, Xen also required changes that would
-perform the dom0 kernel and initrd measurements properly. As a quick reminder,
+perform the Dom0 kernel and initrd measurements properly. As a quick reminder,
```

Comment:
Because one of the requirements has Dom0 with capital letter:

`Integrate TPM 2.0 software stack into Qubes OS Dom0`

Had to change it for the sake of consistence

Reply:
I hardly find requirements defined in MoU as source of truth on that matter.

Reply:
7726bb3ead3040889f32aa4274105c76061f8632